yt_dlp/extractor/redgifs.py

```diff
@@ -98,10 +103,12 @@ class RedGifsIE(RedGifsBaseInfoExtractor):
             'upload_date': '20211107',
             'uploader': 'ignored52',
             'duration': 16,
-            'view_count': int,
+            'view_count': None,
```

Comment:
```suggestion
            'view_count': int,
```
If you edit tests when there is an issue in the extractor, what's the point of tests?